### PR TITLE
Blazor 3.1 Preview Updates

### LINKED ISSUE #15110: Blazor 3.1 Preview Updates
.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c07e57dc-526f-9479-4695-de7d8f06eb9a
* Version Independent ID: 8226ed72-69b2-3c93-a931-218bd11cf9c8
* Content: [Get started with ASP.NET Core Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/get-started?view=aspnetcore-3.0&tabs=visual-studio)
* Content Source: [aspnetcore/blazor/get-started.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/get-started.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #15110

* Get Started update.
* INCLUDE update.
* Updating dates on the topics that use the INCLUDE.

### CODE REVIEW

File: aspnetcore/blazor/get-started.md [hunk 1]
```diff
@@ -16,12 +16,15 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 Get started with Blazor:
 
-1. Install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
+1. Install tooling:
 
-1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell:
```
Comment (danroth27) [inline aspnetcore/blazor/get-started.md]: I think we should leave this step as is for the 3.0 doc set, but update this step to install 3.1 Preview for the 3.1 Preview doc set. In the 3.0 doc set we can then handle installing 3.1 Preview as part of the steps to install Blazor WebAssembly.

File: aspnetcore/blazor/get-started.md [hunk 2]
```diff
@@ -16,12 +16,15 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
...
+   * For an experience in Blazor Server, download and install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
+   * For an experience in Blazor WebAssembly, download and install the [.NET Core 3.1 Preview SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) and [Visual Studio 16.4 Preview 2 or later](https://visualstudio.microsoft.com/vs/preview/).
+
+1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell. The [Microsoft.AspNetCore.Blazor.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.Templates/) package has a preview version while Blazor WebAssembly is in preview.
```
Comment (danroth27) [inline aspnetcore/blazor/get-started.md]: In the 3.0 doc set, installing the Blazor WebAssembly has two sub steps: install .NET Core 3.1 and then run the command to install the template.

Comment (danroth27) [reply aspnetcore/blazor/get-started.md]: We also need to update the Visual Studio specific steps in the 3.0 doc set to clarify that if you're using Blazor WebAssembly, then you need to install the latest VS 2019 16.4 preview. in the 3.1 doc set we need to tell folks using both Blazor Server and Blazor WebAssembly to VS 2019 16.4 preview.

File: aspnetcore/includes/blazorwasm-preview-notice.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
 > [!IMPORTANT]
 > **Blazor WebAssembly in preview**
 >
-> *Blazor Server* is supported in ASP.NET Core 3.0. *Blazor WebAssembly* is in preview.
```
Comment (danroth27) [inline aspnetcore/includes/blazorwasm-preview-notice.md]: How about we just update the current notice to say "*Blazor Server* is supported in ASP.NET Core 3.0. *Blazor WebAssembly* is in preview for ASP.NET Core 3.1"

**Comment by guardrex** [comment]
@danroth27 Try it now.

File: aspnetcore/blazor/get-started.md [hunk 4]
```diff
@@ -16,6 +16,111 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 Get started with Blazor:
 
+::: moniker range=">= aspnetcore-3.1"
```
Comment (danroth27) [inline aspnetcore/blazor/get-started.md]: The content for 3.1 looks great!

File: aspnetcore/blazor/get-started.md [hunk 5]
```diff
@@ -16,6 +16,111 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
...
+
+   ---
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-3.1"
+
 1. Install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
 
 1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell:
```
Comment (danroth27) [inline aspnetcore/blazor/get-started.md]: For the 3.0, we still want to give updated instructions on how to install Blazor WebAssembly using .NET Core 3.1. This seems like the most natural spot to do that. We should update this step to tell folks to 1. Install .NET Core 3.1 Preview, and 2. run the command to install the latest version of the Blazor WebAssembly templates (3.1.0-preview1.blah). Something like this:

```
1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by installing the [.NET Core 3.1 Preview SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) and then running the following command in a command shell:
```

In the VS section of the 3.0 docs we also need to tell folks using Blazor WebAssembly to install VS 2019 16.4.

**Comment by guardrex** [comment]
@danroth27 ...... I should let Merlin ... our family 🐈 give it a try .... I think he'd do a better job! :smile: lol

File: aspnetcore/blazor/get-started.md [hunk 6]
```diff
@@ -30,6 +135,8 @@ Get started with Blazor:
 
    1\. Install the latest [Visual Studio](https://visualstudio.com/vs/) with the **ASP.NET and web development** workload.
 
+       Optionally install [Visual Studio 16.4 Preview 2 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload for Blazor WebAssembly app development.
```
Comment (danroth27) [inline aspnetcore/blazor/get-started.md]: For some reason this is rendering as a code block on the review site:

image

Comment (guardrex) [reply aspnetcore/blazor/get-started.md]: I'll take a look.

Comment (guardrex) [reply aspnetcore/blazor/get-started.md]: I'm going to give that a dedicated list number ... that hopefully will fix it. 🤞 Stand-by ....

**Comment by guardrex** [comment]
Try it now ............ to put a numbered list inside of a numbered list in CommonMark, we have to force the inner list to use numbers with this syntax: `1\. Item, 2\. Item, ...`. See if that last commit fixed it.

**Pull request merged**